tajo-core/src/main/java/org/apache/tajo/engine/parser/SQLAnalyzer.java

```diff
@@ -491,14 +506,16 @@ public WindowSpec buildWindowSpec(SQLParser.Window_specificationContext ctx) {
     WindowFrameStartBoundType boundType = null;
     if (checkIfExist(context.UNBOUNDED())) {
       boundType = WindowFrameStartBoundType.UNBOUNDED_PRECEDING;
-    } else if (checkIfExist(context.unsigned_value_specification())) {
+    } else if (checkIfExist(context.PRECEDING())) {
       boundType = WindowFrameStartBoundType.PRECEDING;
+    } else if (checkIfExist(context.FOLLOWING())) {
+      boundType = WindowFrameStartBoundType.FOLLOWING;
     } else {
       boundType = WindowFrameStartBoundType.CURRENT_ROW;
```

Comment:
I found another initialization code for the implicit window frame in ```ExprAnnotator::visitWindowFunction```. I wonder how this line is related to that code.